helm/idol-omnigroupserver/resources/001_oauth_startup.sh

```diff
@@ -0,0 +1,11 @@
+#/bin/bash
```

Comment:
Add copyright notice

Reply:
(and to other new files)

Reply:
[1be94ca](https://github.com/opentext-idol/idol-containers-toolkit/pull/31/commits/1be94cafeb92b5b4dadb3d5727d24c42d9203624)